packages/shared/src/loaders/deriv-reports-loader.js

```diff
@@ -0,0 +1,47 @@
+const resolve = require('path').resolve;
```

Comment:
We should not be importing reports in the shared package. This will lead to circular dependecy.
Shared is supposed to contain only utility functions.

Reply:
we are not importing the reports in shared package, this loader is just preparing us to have access to reports component out of the reports package( we need to import `PositionsDrawerCard` component in trader) and we have same thing for account , cashier , ....